Adds card liking button to cards

vipulroxx opened the pull request:
Adds like button and fixes word wrapping in the card container.
Screenshot:
![changesvotebranch](https://user-images.githubusercontent.com/6532730/32124920-543abdc6-bb2f-11e7-888e-1169527b5559.png)

- Fixes the issue when the string in the card was larger than the width of the card.
- Added a button to render the like.
- Like button is disabled after it is clicked and the global state of total number of like of the particular card increments by 1.
- Changed the position of the button to be aligned at the end of the card.

This closes #8.

vipulroxx committed:
Fixes word wrapping issue and adds a like instance to the card component.

- Fixes the issue when the string in the card was larger than the width of the card.
- Added a button to render the like.
- Like button is disabled after it is clicked and the global state of total number of like of the particular card increments by 1.
- Changed the position of the button to be alligned at the end of the card.

floogulinc reviewed (changes requested):
This needs testing for the added feature and the existing testing needs to be fixed. Also this doesn't update that the card was liked on the database.

vipulroxx committed:
Added tests for liking.

- Checks whether the button is getting clicked.
- Checks whether the likes are getting incremented after it is clicked.
